backend/de.metas.contracts/src/main/java/de/metas/contracts/modular/log/ModularContractLogDAO.java

```diff
@@ -128,6 +132,16 @@ public ModularContractLogEntryId create(@NonNull final LogEntryCreateRequest req
 			InterfaceWrapperHelper.setValue(log, request.getSubEntryId().getColumnName(), request.getSubEntryId().getId().getRepoId());
 		}
 
+		Optional.ofNullable(request.getPriceActual())
```

Comment:
hmmm, if you would like to go the Optionals way, 
IMHO request.getPriceActual() shall return Optional.